### DISCUSSION TITLE
Error when calling submitFeedAsync

### DISCUSSION THREAD

**Discussion by futureAlec** [abuzuhri/Amazon-SP-API-CSharp#415] [General]
Hi all, has anyone else encountered this error when submitFeedAsync gets called? (I do not even have system.text.json referenced)

{"Method not found: 'Void System.Text.Json.JsonSerializerOptions..ctor(System.Text.Json.JsonSerializerDefaults)'."}

Any help would be appreciated.

**Comment by abuzuhri** [comment]
Please share code you try to use to check from outside

**Comment by futureAlec** [comment]
This is the actual call. Connection and credentials are all built up and good.

var feedID = amazonConnection.Feed.SubmitFeed(xml, FeedType.POST_INVENTORY_AVAILABILITY_DATA,marketPlaceId, options);

xml is valid, as well as marketplaceid list
